### ISSUE TITLE
Is it possible to schedule a task from inside another?

### ISSUE THREAD

**Issue by ImranR98** [fluttercommunity/flutter_workmanager#428]
I've tried scheduling a one-off task from inside an existing periodic task (using `Workmanager().executeTask`) but it looks like the second task is not being run. Is this possible? I assumed from reading #371 that this might be

**Comment by AdamBridges** [comment]
Yes, it should be as long as the one-off task isn't using the same `uniqueName` as the periodic task.

Also, take note that periodic tasks only work for Android and any inputData with a List that's being passed needs to have its subtype redeclared (#426).
